packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -124,6 +125,7 @@ library TaikoData {
         // this block is not verified as the last block in a batch, verifiedTransitionId
         // will remain zero.
         uint32 verifiedTransitionId;
+        uint64 l1StateBlockNumber;
```

Comment:
What's the meaning of l1StateBlockNumber?

Reply:
The L1 block that will be used in the anchor tx. So the L1 block for which the state will be available on L2. It's the name I also used in raiko, haven't though of a better one.

Reply:
Maybe add a comment for it, something like `l1StateBlockNumber will be the value for _l1BlockId in TaikoL2's anchor() function`.